app/src/main/java/com/cdio/solitaire/controller/StrategyController.kt

```diff
@@ -210,80 +212,259 @@ object StrategyController {
                     }
                 }
             }
-
         }
         return null
     }
 
-    fun getAGoodMoveQueueFromTalonAndStock() {
-        var foundAGoodMoveInTalon = false
-        var distanceFromHeadTalon = 0
-        var foundGoodMoveInStock = false
-        var distanceFromHeadStock = 0
-        //If Stock and Talon are completely discovered.
-        if (gsc.gameState.talon.head != null) {
-            var tempTalonCard = gsc.gameState.talon.head
-            while (tempTalonCard != null) {
-                var foundation = gsc.getFoundation(tempTalonCard.suit)
-                if (foundation.size == 0 && tempTalonCard.rank == Rank.ACE) {
-                    foundAGoodMoveInTalon = true
-                    break
-                    //TempTalon to Foundation is now a good move.
-                } else if (foundation.size != 0 && foundation.tail!!.rank.ordinal == tempTalonCard.rank.ordinal - 1) {
-                    foundAGoodMoveInTalon = true
-                    break
-                    //TempTalon to Foundation is now a good move.
+
+    fun modolusThreeOfTalonAndStock(): Int {
+        if ((gsc.gameState.talon.size + gsc.gameState.stock.size) > 3) {
+            return (gsc.gameState.talon.size + gsc.gameState.stock.size) % 3
+        }
+        return 0
+    }
```

Comment:
Function is never used.

Also, should probably be named `moduloThreeOfTalonAndStock`, instead.